kobo/django/auth/migrations/0002_LongnameUser.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from south.db import db
+from south.v2 import SchemaMigration
+
+class Migration(SchemaMigration):
```

Comment:
Can you clarify the purpose of this... is this needed because 6fadd7fb7053277931965abae156713a9ef9c10a renamed the User class but didn't include any migration?